src/views/Customer/About.js

```diff
@@ -0,0 +1 @@
+import React from './node_modules/react';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['React' is defined but never used.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5592511)